src/Api/Serializer/RestSerializer.php

```diff
@@ -250,4 +268,25 @@ private function hasPayloadParam(StructureShape $input, $payload)
         }
         return false;
     }
+
+    private function appendQuery($query, $endpoint)
+    {
+        $append = Psr7\Query::build($query);
+        return $endpoint .= strpos($endpoint, '?') ? "&{$append}" : "?$append";
+    }
+
+    private function getVarspecs($command, $args)
```

Comment:
what does varspecs stand for?  Variable specifications?  Feels kind of like a confusing name

Reply:
Agreed. pretty confusing.  I just kept the variable name from the original implementation, but will change that